mod-calendar/src/main/resources/bama/mod-calendar/features/calendarPeriods.feature

```diff
@@ -341,30 +337,7 @@ Feature: Calendar periods
     Given path 'calendar/periods/' + servicePointId + '/period/' + periodId
     And request updatedInvalidPeriodRequest
     When method PUT
-    Then status 422
-    And match $.errors[0].message == 'must not be null'
-    And match $.errors[0].parameters[0].key == 'startDate'
-
-  Scenario: PUT library hours period by id should return 422 if periods overlap
```

Comment:
The `PUT` endpoint originally never supported creating a new period, only updating one.  If a new calendar was provided, it would be validated like a request to the POST endpoint, however, silently discard it and return a `204`.